### PR TITLE
fixing an issue that causes backup to fail if there exists folders for dropped tables 

### PR DESCRIPTION
**Pull request by VinceLu**
fixing the issue which allows the logic to enter Column Family folder that belongs to a dropped table when iterating snapshots to upload.

### CODE REVIEW

File: cassandra-executor/src/main/java/com/mesosphere/dcos/cassandra/executor/backup/StorageUtil.java [hunk 1]
```diff
@@ -26,8 +26,8 @@
   /**
    * Filters unwanted keyspaces and column families
    */
-  static boolean isValidBackupDir(File ksDir, File cfDir, File bkDir) {
-    if (!bkDir.isDirectory() && !bkDir.exists()) {
+  static boolean isValidBackupDir(File ksDir, File cfDir, File ssDir, File bkDir) {
+    if (!ssDir.exists() || !ssDir.isDirectory() || !bkDir.exists()) {
```
Comment (vargup) [inline cassandra-executor/src/main/java/com/mesosphere/dcos/cassandra/executor/backup/StorageUtil.java]: you should check  ! bkDir.isDirectory() ?

Comment (VinceLu) [inline cassandra-executor/src/main/java/com/mesosphere/dcos/cassandra/executor/backup/StorageUtil.java]: check is added.

**Comment by VinceLu** [comment]
retest this please

**Comment by vargup** [comment]
retest this please

**Comment by vargup** [comment]
LGTM.

**Pull request merged**